factors.py

```diff
@@ -0,0 +1,6 @@
+num=input("Enter number : ")
+count=1
+while count<=num:
+  if num%count==0:
+    print count,
+  count+=1
```

Comment:
This can be optimized.

No need to run the loop from 1 to num. Need to run it only till num/2, add 1 to final result, which is the number itself